foremast-metrics/foremast-spring-boot-k8s-metrics-starter/.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/*
```

Comment:
does it make more sense to have a `.gitignore` file at this directory depth OR would it be even better to have the `.gitignore` closer to the root of the git repo?

the front-end portion of the project has a `.gitignore` file in `foremast-browser/` directory which could be combined with another if brought up to root of repo

Reply:
Multiple gitignore should be fine